lib/fxpay/index.js

```diff
@@ -69,14 +69,13 @@
     }
 
     var partialProdInfo = {productId: productId};
-    var log = settings.log;
 
     if (!onPurchase) {
       onPurchase = function _onPurchase(err, returnedProdInfo) {
         if (err) {
           throw err;
         }
-        console.log('product', returnedProdInfo.productId, 'purchased');
+        settings.log('product', returnedProdInfo.productId, 'purchased');
```

Comment:
On Wed, Jan 28, 2015 at 11:01 AM, Stuart Colville <notifications@github.com>
wrote:

> In lib/fxpay/index.js
> <https://github.com/mozilla/fxpay/pull/66#discussion_r23702297>:
>
> >
> >      if (!onPurchase) {
> >        onPurchase = function _onPurchase(err, returnedProdInfo) {
> >          if (err) {
> >            throw err;
> >          }
> > -        console.log('product', returnedProdInfo.productId, 'purchased');
> > +        settings.log('product', returnedProdInfo.productId, 'purchased');
>
> settings.log.log is not easy on the eye!
>

Yeah. I try to use settings.log.info()


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/fxpay/pull/66/files#r23702297>.
>